identity-service/src/notifications/formatNotificationMetadata.js

```diff
@@ -214,21 +224,54 @@ const TrendingTrackTitle = 'Congrats - You’re Trending! 📈'
 const RemixCreateTitle = 'New Remix Of Your Track ♻️'
 const RemixCosignTitle = 'New Track Co-Sign! 🔥'
 
+const challengeInfoMap = {
+  'profile-completion': {
+    title: '✅️ Complete your Profile',
+    amount: 5,
+  },
+  'listen-streak': {
+    title: '🎧 Listening Streak: 7 Days',
+    amount: 5,
+  },
+  'track-upload': {
+    title: '🎶 Upload 5 Tracks',
+    amount: 5,
+  },
+  'referrals': {
+    title: '📨 Invite your Friends',
+    amount: 1,
+  },
+  // TODO: Confirm copy for this challenge
+  'referred': {
+    title: '📨 Invite your Friends',
+    amount: 1,
+  },
+  'connect-verified': {
+    title: '✅️ Link Verified Accounts',
+    amount: 10,
+  },
+  'mobile-install': {
+    title: '📲 Get the App',
+    amount: 10,
+  },
+}
+
```

Comment:
This isn't great to have a map here and in client to do a similar thing. Not sure what a better way to handle this would be